docs/upgradeFromRC3.md

```diff
@@ -0,0 +1,325 @@
+
+### Step 1:
+
+Shut down your `docker-compose` ensemble. Move to your `archipelago-deployment` folder and run this:
+
+```shell
+docker-compose down
+```
+
+### Step 2:
+
+Verify that all containers are actually down. The following command should return an empty listing. If anything is still running, wait a little longer, and run the following comman again.
+
+```shell
+docker ps
+```
+
+### Step 3:
+
+Now let's `tar.gz` the whole ensemble with data and configs. As an example we will save this into your `$HOME` folder.
+As a good practice we append the **current date **(YEAR-MONTH-DAY) to the filename. Here we assume today is August 3rd of 2022.
+
+```shell
+cd ..
+sudo tar -czvpf $HOME/archipelago-deployment-RC3-backup-20220803.tar.gz archipelago-deployment
+cd archipelago-deployment
+```
+
+The process may take a few minutes. Now let's verify that all is there and that the `tar.gz` is not corrupt.
+
+```shell
+tar -tvvf $HOME/archipelago-deployment-RC3-backup-20220803.tar.gz 
+```
+
+You will see a listing of files. If corrupt (Do you have enough space? Did your ssh connection drop?) you will see the following:
+
+```shell
+tar: Unrecognized archive format
+```
+
+### Step 4:
+
+Restart your `docker-compose` ensemble, and wait a little while for all to start.
+
+```shell
+docker-compose up -d
+```
+
+### Step 5:
+
+Export/backup all of your live Archipelago configurations (this allows you to compare/come back in case you lose something custom during the upgrade).
+
+```shell
+docker exec esmero-php mkdir config/backup
+docker exec esmero-php drush cex --destination=/var/www/html/config/backup
+```
+
+Good. Now it's safe to begin the upgrade.
+
+___
+
+## Upgrading to 1.0.0
+
+### Step 1:
+
+First we are going to disable modules that are not part of 1.0.0 or are not yet compatible with Drupal 9.4.x or higher. Run the following command:
+
+```shell
+docker exec esmero-php drush pm-uninstall search_api_solr_defaults entity_reference
+```
+
+From inside your `archipelago-deployment` repo folder we are going to open up the file `permissions` for some of your most protected Drupal files.
+
+```shell
+sudo chmod 777 web/sites/default
+sudo chmod 666 web/sites/default/*settings.php
+sudo chmod 666 web/sites/default/*services.yml
+```
+
+If on a Linux host we are going to open up these additional permissions:
+
+```shell
+sudo chown -R $USER:$USER web/sites/default/files/color
+sudo find web/sites/default/files/color -type d -exec chmod 755 {} + 
+```
+
+### Step 2:
+
+Time to fetch the `1.0.0` branch and update our `docker-compose` and `composer` dependencies. We are also going to stop the current `docker` ensemble to update all containers to newer versions:
+
+```shell
+docker-compose down
+```
+
+
+```shell
+git checkout 1.0.0 
+```
+
+Then copy the appropriate `docker-compose` file for your architecture:
+
```

Comment:
Hello @aksm --I think it may be helpful here to include the Intel / M1 references for users, as noted in the archipelago-deployment docs.

Reply:
Hi @alliomeria, sounds good. Done here: 06ccf5d5e4e96161c653194f114fa15cf571fa09

Reply:
Thanks @aksm, reads great. We received a couple direct questions about this in the past, so it's good to have in reference now.


@DiegoPino ok to merge now?

Reply:
@alliomeria you are the boss here 🎩  You merge when you feel it is right to merge. Please approve first (that way we give that workflow a try as future good practice).

Reply:
No boss, a collaborative 🐝 ! Ok merging!